docs/source/install.rst

```diff
@@ -63,10 +64,21 @@ packages are specific to different operating systems:
 * macOS (OS X): `MacTeX <http://tug.org/mactex/>`_.
 * Windows: `MikTex <http://www.miktex.org/>`_
 
-Because nbconvert depends on packages and fonts included in standard 
-TeX distributions, if you do not have a complete installation, you 
-may not be able to use nbconvert's standard tooling to convert 
-notebooks to PDF. 
+Because nbconvert depends on packages and fonts included in standard
+TeX distributions, if you do not have a complete installation, you
+may not be able to use nbconvert's standard tooling to convert
+notebooks to PDF.
+
+Installing Chromium
+-------------------
+
+When converting to PDF with ``--to webpdf``, nbconvert uses `Puppeteer <https://pptr.dev>`_, and
+more precisely its Python bindings `pyppeteer <https://github.com/pyppeteer/pyppeteer>`_. pyppeteer
+needs a special version of Chromium, and can automatically download it. Since the download size is
+quite large (around 100 MB), nbconvert prevents it by default. If pyppeteer finds a suitable
+version of Chromium on your system, it will try and use it. Otherwise, please use the
```

Comment:
I wouldn't even advertise it working with any existing chrom*.

Reply:
Good point since that was changed since first discussed.